api-server/src/server/boot/authentication.js

```diff
@@ -188,3 +194,53 @@ function createGetPasswordlessAuth(app) {
     );
   };
 }
+
+function mobileLogin(app) {
+  const {
+    models: { User }
+  } = app;
+  return async function getPasswordlessAuth(req, res, next) {
+    const {
+      headers: { token }
+    } = req;
+
+    const auth0Res = await fetch(
+      `https://${process.env.AUTH0_DOMAIN}/userinfo`,
+      {
+        headers: { Authorization: `Bearer ${token}` }
+      }
+    );
+
+    if (!auth0Res.ok) {
+      return next(
+        wrapHandledError(new Error('Invalid Auth0 token'), {
+          type: 'danger',
+          message: 'We could not log you in, please try again in a moment.',
+          status: auth0Res.status
+        })
+      );
+    }
+
+    const { email } = await auth0Res.json();
+
+    if (!isEmail(email)) {
+      return next(
+        wrapHandledError(new TypeError('decoded email is invalid'), {
+          type: 'danger',
+          message: 'The email is incorrectly formatted',
+          status: 400
+        })
+      );
+    }
+
+    return User.findOne$({ where: { email } })
```

Comment:
```suggestion
    User.findOne$({ where: { email } })
```
This implies it meaningfully returns something, but it doesn't.  Express middlewares only interact via res, req or next - the return value is ignored.